src/platform/windows/display_vram.cpp

```diff
@@ -346,6 +356,12 @@ namespace platf::dxgi {
       // Release encoder mutex to allow capture code to reuse this image
       img_ctx.encoder_mutex->ReleaseSync(0);
 
+      ID3D11RenderTargetView *emptyRenderTarget = nullptr;
+      device_ctx->OMSetRenderTargets(1, &emptyRenderTarget, nullptr);
+      device_ctx->RSSetViewports(0, nullptr);
```

Comment:
I don't think we need to reset the render targets or the viewport here. Those are associated with this `hwdevice_t` rather than the provided image.

Reply:
I can remove it, if you think it can have negative effects on performance. Just having contexts clean of dangling references can save us debugging headache down the line, when in a year or so we decide to refactor or rewrite something.

Reply:
Double checked, and yes it does make sense, Pushed a fix.